src/index.css

```diff
@@ -4,3 +4,7 @@
   --color-brandBeige: #f4f3ef;
   --color-brandPink: #ee9964;
 }
+
+html {
+  scroll-behavior: smooth;
```

Comment:
What's the rationale for this? I really don't like it when pages hijack normal browser behaviour or override settings. Smooth scrolling is something users can toggle on/off themselves in their browser settings.

There are exceptions though, such as to make certain animations or browser-based games behave.